From: Michael Vogt <mvo@debian.org>
Date: Sat, 30 Aug 2014 21:40:04 +0000
Subject: Search the pre-multiarch paths too

needed only until all packages are moved to mutliarch paths

Forwarded: not-needed
---
 girepository/girepository.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/girepository/girepository.c b/girepository/girepository.c
index a0754f4..28aee6d 100644
--- a/girepository/girepository.c
+++ b/girepository/girepository.c
@@ -221,6 +221,9 @@ init_globals (void)
 
       typelib_search_path = g_slist_prepend (typelib_search_path, typelib_dir);
 
+      // compat with pre-multiarch
+      typelib_search_path = g_slist_prepend (typelib_search_path, "/usr/lib/girepository-1.0");
+
       typelib_search_path = g_slist_reverse (typelib_search_path);
     }
 
